guides/common/modules/snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc

```diff
@@ -1,7 +1,2 @@
 * If you want to use {SmartProxyServers} instead of your {ProjectServer}, ensure that you have configured your {SmartProxyServers} accordingly.
-ifdef::orcharhino[]
-For more information, see xref:configuring-{smart-proxy-context}-for-host-registration-and-provisioning_{context}[].
```

Comment:
I thought you weren't using the Installation guides in orcharhino, so I've also added the xref'ed module in the `master.adoc` files in Managing Hosts and Provisioning Hosts (?). Just FYI.

Reply:
If you use the Installing guide, then it might be worth removing the module from the `master.adoc` files as well.